mythril/laser/ethereum/state.py

```diff
@@ -1,16 +1,44 @@
-from z3 import BitVec, BitVecVal, Solver, ExprRef, sat
+from z3 import BitVec, BitVecVal, Solver, ExprRef, Concat, sat, simplify
 from mythril.disassembler.disassembly import Disassembly
 from copy import copy, deepcopy
 from enum import Enum
 from random import randint
 
 from mythril.laser.ethereum.evm_exceptions import StackOverflowException, StackUnderflowException
 
-
 class CalldataType(Enum):
     CONCRETE = 1
     SYMBOLIC = 2
 
+class SymbolicCalldata:
```

Comment:
Can we rename this to Calldata, and implement logic for both concrete + symbolic calldata.
That way we can do away with the calldatatype